site/src/pages/components/invokers.explainer.mdx

```diff
@@ -294,7 +294,7 @@ Invokers, the names and exact semantics may be subject to change:
 | `<details>`     | `'toggle'`            | If the `<details>` is `open`, then close it, otherwise open it                       |
 | `<details>`     | `'open'`              | If the `<details>` is not `open`, then open it                                       |
 | `<details>`     | `'close'`             | If the `<details>` is `open`, then close it                                          |
-| `<dialog>`      | `'toggle'`            | If the `<dialog>` is `open`, cancel the dialog                                       |
+| `<dialog>`      | `'toggle'`            | If the `<dialog>` is `open`, then close it, otherwise call `showModal()`                                      |
```

Comment:
I can't remember what we decided for cancel vs close I believe it should be close and showModal right?